autoload/rainbow_csv.vim

```diff
@@ -820,12 +818,13 @@ func! rainbow_csv#align_field(field, is_first_line, max_field_components_lens, i
     return repeat(' ', integer_delta_length) . clean_field . trailing_spaces
 endfunc
 
-
-func! rainbow_csv#csv_align()
+func! rainbow_csv#csv_align() range
+    let l:firstline = a:firstline
```

Comment:
Please don't use `l:` prefix for consistency with other code.

Reply:
updated